xds/internal/test/xds_server_integration_test.go

```diff
@@ -39,10 +39,12 @@ import (
 	v3tlspb "github.com/envoyproxy/go-control-plane/envoy/extensions/transport_sockets/tls/v3"
 	wrapperspb "github.com/golang/protobuf/ptypes/wrappers"
 	"github.com/google/uuid"
-	xds2 "google.golang.org/grpc/internal/xds"
 	"google.golang.org/protobuf/proto"
 	"google.golang.org/protobuf/types/known/anypb"
 
+	"google.golang.org/grpc/internal/testutils"
```

Comment:
Can you reorganize these imports?

Usually we do:
```
Standard packages
<blank line>
Almost everything else
<blank line>
Protos and, sometimes, renamed imports
```

Reply:
Intellij sometimes messes with these imports way too much. Not sure if I have to change any of my settings.